docs/gemfile

```diff
@@ -1,5 +1,5 @@
 source 'https://rubygems.org'
 
-gem 'kramdown', '~> 2.2', '>= 2.2.1'
+gem 'kramdown', '>= 2.3.1'
```

Comment:
According to https://rubygems.org/gems/kramdown/versions/2.3.0, there is no 2.3.1 version, just 2.0.

Reply:
Yup, typo. Fixed